src/pybamm/models/submodels/particle_mechanics/base_mechanics.py

```diff
@@ -79,7 +149,7 @@ def _get_mechanical_results(self, variables):
         stress_r_surf = pybamm.Scalar(0)
         # Ai2019 eq [8] with r=R
         # c_s_rav is already multiplied by 3/R^3 inside r_average
-        stress_t_surf = Omega * E0 / 3.0 / (1.0 - nu) * (c_s_rav - c_s_surf)
+        stress_t_surf = Omega * E0 * (c_s_rav - c_s_surf) / 3.0 / (1.0 - nu)
```

Comment:
Based on your last test fix, it looks like this is probably right, tests pass for me too
```suggestion
        stress_t_surf = Omega * E0 * (c_s_rav - c_s_surf) * (1.0 - nu) / 3.0
```